extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/handler/LifxLightHandler.java

```diff
@@ -864,8 +869,7 @@ public void handlePowerStatus(StatePowerResponse packet) {
             updateState(CHANNEL_COLOR, currentColorState);
             updateState(CHANNEL_BRIGHTNESS, currentColorState.getBrightness());
         } else {
-            updateState(CHANNEL_COLOR,
-                    new HSBType(currentColorState.getHue(), currentColorState.getSaturation(), PercentType.HUNDRED));
+            updateState(CHANNEL_COLOR, HSBType.WHITE);
```

Comment:
You are changing the functionality here. Before, what was happening is that the brightness was set to 100%, keeping hue and saturation values as before. With your code, your are always setting the color to white instead, which is imho not correct.